ch_tools/chadmin/cli/zookeeper_group.py

```diff
@@ -266,3 +267,20 @@ def delete_ddl_task_command(ctx, tasks):
     """
     paths = [f"/clickhouse/task_queue/ddl/{task}" for task in tasks]
     delete_zk_nodes(ctx, paths)
+
+
+@zookeeper_group.command(name="clickhouse-hosts-cleanup")
+@option("-f", "--fqdn", type=str, help="Removed FQDNs, comma separated", required=True)
+@option(
+    "-c",
+    "--root",
+    "zk_root_path",
+    type=str,
+    help="Cluster ZooKeeper root path. If not specified,the root path will be used.",
+    required=False,
+    default="/",
+)
```

Comment:
The parameter looks redundant. It can be obtained from ClickHouse config file automatically.

Reply:
Summary from the chat:
Move the `--root` knob to zookeeper group.

Reply:
Let's name the option `--chroot` to be consistent with existing option `--no-chroot`.